frontend/app/mappers/benefit-application-service-mappers.server.ts

```diff
@@ -199,15 +253,54 @@ function toInsurancePlan({ hasFederalBenefits, federalSocialProgram, hasProvinci
   return [{ InsurancePlanIdentification: insurancePlanIdentification }];
 }
 
+interface RelatedPerson {
```

Comment:
Remove this interfance since we want to have implicit return types instead of explicit.

Reply:
Removed and reworked RelatedPerson type/functions